lib/jekyll-import/importers/blogger.rb

```diff
@@ -183,20 +189,26 @@ def tag_end(tag)
                 target_dir = '_drafts' if @in_entry_elem[:meta][:draft]
 
                 FileUtils.mkdir_p(target_dir)
-      
-                File.open(File.join(target_dir, "#{post_data[:filename]}.html"), 'w') do |f|
+
+                extension = markdown ? 'md' : 'html'
+                file_name = URI::decode("#{post_data[:filename]}.#{extension}")
+                File.open(File.join(target_dir, file_name), 'w') do |f|
                   f.flock(File::LOCK_EX)
-      
+
                   f << post_data[:header].to_yaml
                   f << "---\n\n"
-                  f << post_data[:body]
+                  if markdown
+                    f << ReverseMarkdown.convert(post_data[:body])
```

Comment:
I see the problem with the tests. Please add the require above this line, like so:

```ruby
if markdown
  JekyllImport.require_with_fallback(%w[reverse_markdown])
  f << ReverseMarkdown.convert(post_data[:body])
else 
  f << post_data[:body]
end
```

If it's in the `require_deps`, it's required even if you don't want to use ReverseMarkdown.